tests/integration/itest

```diff
@@ -274,8 +286,18 @@ def run_test(args, target, parser):
             subprocess.check_call(['with-db', 'bltoolsdevbq-bq_itest',
                                    'nosetests', '--xunit-file=nosetests.xml', '.'],
                                   cwd=f"{script_dir}/records/single_db")
+    elif (target == 'bigquery-gcs'):
+        with set_gcs_scratch_bucket():
+            if (args.docker):
+                docker_compose_run(['with-db', 'bltoolsdevbq-bq_itest',
+                                    'nosetests', '--xunit-file=nosetests.xml', '.'],
+                                   cwd="/usr/src/app/tests/integration/records/single_db")
+            else:
+                subprocess.check_call(['with-db', 'bltoolsdevbq-bq_itest',
+                                       'nosetests', '--xunit-file=nosetests.xml', '.'],
+                                      cwd=f"{script_dir}/records/single_db")
```

Comment:
It also adds the ability to integration test BigQuery with and without GCS for export.